packages/lisk-transactions/src/0_transfer.js

```diff
@@ -27,12 +51,11 @@ const createAsset = data => {
 			throw new Error(
 				'Invalid encoding in transaction data. Data must be utf-8 encoded.',
```

Comment:
I'm not actually confident we should enforce UTF8.

Reply:
@shuse2 could you please let me know what do you think on this? I have no idea :D

Reply:
@shuse2 @willclarktech could you elaborate this?

Reply:
Issue created: https://github.com/LiskHQ/lisk-elements/issues/771